### ISSUE TITLE
[Question]modify the initial config

### ISSUE THREAD

**Issue by GarryJAY502** [MIC-DKFZ/nnDetection#282]
### :question: Question

Hi,
We had a question about the config file.
How should I modify the initial epoch batch learnrate and other parameters during training.
which file
Thanks!

**Comment by partha-ghosh** [comment]
Hi @GarryJAY502 ,
If the `config.yaml` you can find :
```
defaults:
    - _self_
    - train: retinaunet_hnm_v002
```
The configuration works by inheriting configuration from `retinaunet_hnm_v002.yaml` in the `train` folder. The same way, you can find the configurations `retinaunet_hnm_v002.yaml` is inheriting from other config files. The trainer-specific configuration is in the `trainer_cfg` folder, where you can update the learning rate, number of epochs etc.

You can also update the parameters through the command line itself. For example, to update the `warm_iterations` to 5000 and `initial_lr` to 0.001, you have to pass `-o trainer_cfg.warm_iterations=5000 trainer_cfg.initial_lr=0.001`. This way you can update other parameters as well.

Best,
Partha

**Comment by GarryJAY502** [comment]
> 你好@GarryJAY502， 如果`config.yaml`您能找到：
> 
> ```
> defaults:
>     - _self_
>     - train: retinaunet_hnm_v002
> ```
> 
> 配置通过从文件夹`retinaunet_hnm_v002.yaml`中继承配置来工作`train`。同样，您可以发现配置`retinaunet_hnm_v002.yaml`是从其他配置文件继承的。训练器特定的配置位于`trainer_cfg`文件夹中，您可以在其中更新学习率、时期数等。
> 
> 您还可以通过命令行本身更新参数。例如，要将更新`warm_iterations`为 5000 和`initial_lr`0.001，您必须传递`-o trainer_cfg.warm_iterations=5000 trainer_cfg.initial_lr=0.001`。这样您也可以更新其他参数。
> 
> 最好的， 帕塔

It works，thanks ,Partha
